Kernel/API/Syscall.h

```diff
@@ -122,7 +122,7 @@ enum class NeedsBigProcessLock {
     S(listen, NeedsBigProcessLock::No)                     \
     S(lseek, NeedsBigProcessLock::No)                      \
     S(madvise, NeedsBigProcessLock::Yes)                   \
-    S(map_time_page, NeedsBigProcessLock::Yes)             \
+    S(map_time_page, NeedsBigProcessLock::No)              \
```

Comment:
Why is this one in its own commit?
I have no opinion on if they should all be in separate commit or in the same, it's only for consistency.

Reply:
If something goes wrong it could be easier to revert this. Also, it's a common thing to split commits for removing the big lock from syscalls by the category of the syscall - for example, memory-related syscalls should not be mixed with time-related syscalls.